src/System.IO.Compression.Brotli/System/IO/Compression/Brotli.cs

```diff
@@ -15,8 +15,67 @@ namespace System.IO.Compression
 {
     public static class Brotli
     {
-        const int DefaultQuality = 11;
-        const int DefaultWindowSize = 24;
+        private const int MinWindowBits = 10;
+        private const int MaxWindowBits = 24;
+        private const int MinQuality = 0;
+        private const int MaxQuality = 11;
+
+        public struct State : IDisposable
+        {
+            internal IntPtr BrotliNativeState { get; private set; }
+
+            public void Dispose()
+            {
+                //    BrotliNative.BrotliDecoderDestroyInstance(BrotliNativeState);
+                //   BrotliNative.BrotliEncoderDestroyInstance(BrotliNativeState);
+            }
+
+            public void InitializeDecoder()
+            {
+                BrotliNativeState = BrotliNative.BrotliDecoderCreateInstance();
+                if (BrotliNativeState == IntPtr.Zero)
+                {
+                    throw new System.IO.IOException(BrotliEx.DecoderInstanceCreate);
+                }
+            }
+
+            public void InitializeEncoder()
+            {
+                BrotliNativeState = BrotliNative.BrotliEncoderCreateInstance();
+                if (BrotliNativeState == IntPtr.Zero)
+                {
+                    throw new System.IO.IOException(BrotliEx.EncoderInstanceCreate);
+                }
+            }
+
+            public void SetQuality(uint quality)
+            {
+                if (quality < MinQuality || quality > MaxQuality)
```

Comment:
MinQuality is 0 and quality is an unsigned int. Quality will never be < MinQuality and hence this check is unnecessary.